client/question/services/FeedbackGeneratorService.js

```diff
@@ -803,6 +803,14 @@ tie.factory('FeedbackGeneratorService', [
                   feedback.appendErrorParagraph(paragraph.content);
                 }
               });
+
+              var errorLineNumber = prereqCheckFailure.getErrorLineNumber();
+              if (errorLineNumber) {
+                var errorStr = '(See line ' + errorLineNumber +
```

Comment:
Optional: Consider just inlining this in line 812 rather than declaring a new var.

Reply:
Updated, will go out with the next commit.